cookbooks/aws-parallelcluster-slurm/recipes/update/update_head_node.rb

```diff
@@ -20,6 +20,8 @@
   not_if { ::File.exist?(node['cluster']['previous_cluster_config_path']) && !are_queues_updated? && !are_bulk_custom_slurm_settings_updated? }
 end
 
+include_recipe 'aws-parallelcluster-slurm::config_slurm_resume'
```

Comment:
Can we add a unit test verifying that the recipe is always called as part of the update?